dashboard/src/components/AppView/ServicesTable/ServiceItem.test.tsx

```diff
@@ -43,6 +53,9 @@ it("renders a view with IP", () => {
     spec: { ports: [{ port: 80 }], type: "LoadBalancer" },
     status: { loadBalancer: { ingress: [{ ip: "1.2.3.4" }] } },
   } as IResource;
-  const wrapper = shallow(<ServiceItem service={service} />);
+  kubeItem.item = service;
+  const wrapper = shallow(
+    <ServiceItem service={kubeItem} name={service.metadata.name} getService={jest.fn()} />,
+  );
   expect(wrapper).toMatchSnapshot();
 });
```

Comment:
We would need to move here the tests to ensure that `"ItBehavesLike"` a loading component and that it renders an error if present.

Reply:
yep, although ItBehavesLike can't be used because we don't use `LoadingWrapper`, but rather use `LoadingSpinner` directly.

Reply:
ah, I didn't notice that. Could you add a comment there explaining why do you use that instead of the LoadingWrapper? (Since we usually use that by default).

Also then here we would need to create more verbose tests just checking if the LoadingSpinner exists,

Reply:
Good shout, and after discussing with @migmartri I've changed this to use LoadingWrapper. Also created https://github.com/kubeapps/kubeapps/issues/925 for ConfirmDialog which is using LoadingSpinner directly. Thanks!